src/scripts/components/stories/story-filter/story-filter.tsx

```diff
@@ -13,18 +13,16 @@ import setSelectedStoryTags from '../../../actions/set-selected-story-tags';
 import styles from './story-filter.styl';
 
 let translateValue = 0;
```

Comment:
when we don't use this in a requestAnimationFrame loop, this value should be in a useState variable and use the normal react render mechanism